designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -133,6 +133,12 @@ class ProcessesResources(
             processService.getProcessActions(processId.id)
           }
         }
+      } ~ path("processes" / ProcessNameSegment / "lastDeployedVersionId") { processName =>
+        processId(processName) { processId =>
+          complete {
+            processService.getLastDeployedVersionId(processId.id)
```

Comment:
Can we use results of `processService.getLatestProcessWithDetails` or `processService.getProcessWithDetails` instead of adding new endpoint? See `ScenarioWithDetails.lastDeployedAction`, there is some info on current and last deployed version.